server/events/project_command_builder.go

```diff
@@ -604,91 +623,9 @@ func (p *DefaultProjectCommandBuilder) buildAllProjectCommandsByPlan(ctx *comman
 	return cmds, nil
 }
 
-// buildProjectApplyCommand builds an apply command for the single project
-// identified by cmd.
-func (p *DefaultProjectCommandBuilder) buildProjectApplyCommand(ctx *command.Context, cmd *CommentCommand) ([]command.ProjectContext, error) {
```

Comment:
`buildProjectApplyCommand`, `buildProjectVersionCommand`, `buildProjectImportCommand` and `buildProjectStateCommand` become same impls. So merge it into `buildProjectCommand`.